hs_core/forms.py

```diff
@@ -286,6 +286,12 @@ class ContributorValidationForm(PartyValidationForm):
     pass
 
 
+class CitationValidationForm(PartyValidationForm):
```

Comment:
Why extend the party validation form?  Citations do not have organization, email, address, phone, etc.

Reply:
This has been removed